apps/cloudfront-functions/src/viewer-request-handler.ts

```diff
@@ -13,8 +13,8 @@ const handler = (
     const { request } = event;
     const uri = request.uri;
 
-    // Add the .html extension if missing
-    if (!uri.endsWith('/') && !/\.[a-zA-Z]+$/.test(uri)) {
+    // Add the .html extension if missing and the uri does not end with .woff2
```

Comment:
```suggestion
    // Add the .html extension if missing
```
I don't think there's a need to specify .woff2. It shows from tests.